crates/story/examples/fixtures/test.kt

```
@@ -0,0 +1,97 @@
+            |Name: $name
+            |Elapsed: ${elapsed}s
+            |Config: timeout=${config.timeout}, retries=${config.retries}
+        """.trimMargin()
+    }
+
+    override fun toString(): String = "HelloWorld(name=$name)"
+}
+
+fun <T> safely(block: () -> T): Result<T> = try {
+    Result.Success(block())
+} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)